ev-dev-tools/src/ev_cli/templates/interface-Exports.hpp.j2

```diff
@@ -81,6 +81,12 @@ public:
         {% elif 'object_type' in cmd.result %}
         json retval_json = result.value();
         {{ result_type(cmd.result) }} retval = retval_json;
+        {% elif 'array_type' in cmd.result %}
+        auto array = {{ var_to_cpp(cmd.result) }}(result.value());
```

Comment:
I think this line here shouldn't be necessary.  It should also work if you use `result.value()` inside the for loop instead of `array`